editor/post-title/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function PostTitle( { title, onUpdate } ) {
+	const onChange = ( event ) => onUpdate( event.target.value );
+
+	return (
+		<h1 className="editor-post-title">
```

Comment:
Yeah, the reasoning was that the post title should be `h1` but I agree this is essential for the frontend but probably not for the editor itself.